requirements-extras.txt

```diff
@@ -360,7 +360,7 @@ coverage==7.8.2 \
     --hash=sha256:ef2f22795a7aca99fc3c84393a55a53dd18ab8c93fb431004e4d8f0774150f54 \
     --hash=sha256:fd51355ab8a372d89fb0e6a31719e825cf8df8b6724bee942fb5b92c3f016ba3
     # via pytest-cov
-createrepo-c==1.2.1 \
+createrepo-c==1.2.1 ; sys_platform == 'linux' \
```

Comment:
I'd like to see `createrepo-c` to be extracted from the core set of deps into its own extra in the future as it's been giving us a hard time for a while, something like `pip install .[rpm]` and keep it isolated in its own requirements file, and keep markers only in that file. This is because `nox` tries to install `requirements.txt` for all sessions, even unit tests which don't need `createrepo_c` at all, it's a runtime dependency only. That said, it's been broken since we added this so this PR doesn't make anything worse in that regard, instead, it improves **some** of the aspects.